### ISSUE TITLE
[easymde] handle image upload

### ISSUE THREAD

**Issue by assertchris** [blade-ui-kit/blade-ui-kit#16]

**Comment by mokhosh** [comment]
also Trix?

**Comment by tnylea** [comment]
Hey guys, I wrote a pretty good tutorial on creating a Trix Editor Livewire component. You could probably use the code in this tutorial to easily add image attachments to the Trix component ;)

[https://devdojo.com/tnylea/laravel-livewire-trix-editor-component](https://devdojo.com/tnylea/laravel-livewire-trix-editor-component)

Thought it would be helpful. Hope you guys have a great day 🙌

**Comment by mokhosh** [comment]
Yeah saw that on laravel news and added it to my reading list. Good job.
